.github/workflows/gosec.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Gosec
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      - name: Run Gosec Security Scanner
+        uses: securego/gosec@master
```

Comment:
but this uses original `gosec`, maybe we need to use `informalsystems/gosec@master` .

Reply:
whoa thank you.  I need to change this, it is from the informal gosec page, a copy/paste.  

<3

Reply:
yes, they need to update it.